nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -373,6 +374,11 @@ protected Collection<ValidationResult> customValidate(ValidationContext context)
         boolean clientIsSet = context.getProperty(CLIENT_SERVICE).isSet();
         boolean uriIsSet    = context.getProperty(URI).isSet();
 
+        if (uriIsSet) {
+            DeprecationLoggerFactory.getLogger(AbstractMongoProcessor.class).warn("The MongoDB URI and other client " +
```

Comment:
The `getLogger()` call should use `getClass()` to reference the actual Processor class instead of the abstract class.
```suggestion
            DeprecationLoggerFactory.getLogger(getClass()).warn("The MongoDB URI and other client " +
```